frontend/next.config.js

```diff
@@ -53,4 +53,5 @@ module.exports = {
 
     return config;
   },
+  webpack5: false,
```

Comment:
This is a temporary solution to run a project. It must be solved in next iterations